sdk/cognitivelanguage/ai-language-conversations/package.json

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "@azure/ai-language-conversations",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "description": "An isomorphic client library for the Azure Conversational Language Understanding service.",
+  "version": "1.0.0-beta.1",
+  "engines": { "node": ">=12.0.0" },
+  "dependencies": {
+    "@azure/core-lro": "^2.2.0",
+    "@azure/abort-controller": "^1.0.0",
+    "@azure/core-client": "^1.5.0",
+    "@azure/core-auth": "^1.3.0",
+    "@azure/core-rest-pipeline": "^1.8.0",
+    "tslib": "^2.2.0"
+  },
+  "keywords": ["node", "azure", "typescript", "browser", "isomorphic"],
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/ai-language-conversations.d.ts",
+  "devDependencies": {
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/identity": "^2.1.0",
+    "@azure/logger": "^1.0.3",
+    "@azure/test-utils": "^1.0.0",
+    "@azure/core-tracing": "1.0.0",
```

Comment:
should this be moved to the runtime deps?

Reply:
Yeah I moved both identity and core-tracing to runtime dependencies.